app/assets/javascripts/findings.js.erb

```diff
@@ -179,6 +179,64 @@ jQuery(function ($) {
     }
   })
 
+  $(document).on('change', '[data-toggle-suggested-impact-disabled-attr]', function () {
+    if ($(this).is(':checked')){
+      $('select[name$="[impact_risk]"]').attr('disabled', true)
+      $('input[name$="[impact_risk]"]').attr('disabled', false)
+
+      var impactValue = $('[data-impact-risks-value]').text()
+
+      $('[name$="[impact_risk]"]').val($.trim(impactValue)).trigger('change')
+    }else{
+      $('select[name$="[impact_risk]"]').attr('disabled', false)
+      $('input[name$="[impact_risk]"]').attr('disabled', true)
+    }
+  })
+
+  $(document).on('change', '[data-finding-issue-amount]', function() {
+    var textImpact
+    var valueImpact
+    var totalAmount = 0
+    var types       = $('[data-impact-risks-types]').data('impactRisksTypes')
+
+    $('[data-finding-issue-amount]').each(function (i, e) {
+      totalAmount += parseFloat($(e).val())
+    })
+
+    $.each(types, function(amount, name) {
+      if (totalAmount < amount) {
+        result      = name.split('|')
+        textImpact  = result[0]
+        valueImpact = result[1]
+        return false
```

Comment:
¿Porqué?

Reply:
Para que tome el primer valor que encuentra

Reply:
¿Estás seguro que each funciona así?

Reply:
No el 100%, pero lo tome segun esto: 
return false; // this is equivalent of 'break' for jQuery loop
return;       // this is equivalent of 'continue' for jQuery loop

Reply:
Si, ahí miré en el API de jQuery y es así. Es el primer each que veo que lo hace, eso suele llamarse find en el resto del mundo.

Reply:
```suggestion

        return false
```

Reply:
No está lo del suggestion.

Reply:
No está de nuevo el suggestion.